pkg/apps/helmops.go

```diff
@@ -47,10 +50,41 @@ func (o *HelmOpsOptions) AddApp(app string, chart string, version string, values
 	if err != nil {
 		return errors.Wrapf(err, "attaching values.yaml to %s", appCRDName)
 	}
+	err = o.addAppMetadata(appCRDName, chart, repository)
+	if err != nil {
+		return err
+	}
 	log.Infof("Successfully installed %s %s\n", util.ColorInfo(app), util.ColorInfo(version))
 	return nil
 }
 
+func (o *HelmOpsOptions) addAppMetadata(name string, chartDir string, repository string) error {
```

Comment:
You should be able to fold this into`StashValues` to avoid refetching the CRD. But that's minor :-)